Bottom: 60e610c3d34ae7a3f8d0125011d868f5cb382da7
Top:    e7686e967fd92cdf5a4e5cc33bfb257e0b801b51
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-03 20:27:12 +0000

find.c: The User varaible needs to die...

Reduce its usage to where absolutely necessary... oh, what a surprise,
another variable passing state down into add_object_to_list ()


---

diff --git a/src/find.c b/src/find.c
index af5aa3e..70cc17f 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3090,7 +3090,7 @@ LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int flag,
    */
   ListStart (type, ptr1, ptr2, ptr3, flag);
   DoIt (flag, AndRats, AndDraw);
-  if (User)
+  if (AndDraw)
     IncrementUndoSerialNumber ();
   User = false;
